Language/Functions/element_type_A02_t03.dart

```diff
@@ -39,7 +41,9 @@ FutureOr<Stream<int>?> foo() async* {
 }
 
 main() async {
+  asyncMultiStart(2);
```

Comment:
This is somewhat confusing, as locally only reading `main` one doesn't know where the second `asyncEnd()` is. I'd really encourage you to follow my suggestion above: Any strait line code that triggers an async operation (e.g. `await`, `.then((_) {})`) should have an `asyncStart` at the beginning and an `asyncEnd` when the operations completes.

Usually there will be balanced `asyncStart` and `asyncEnd` in the same function, making it easily understandable.